### PR TITLE
Updated the documentation

### PR DESCRIPTION
**Pull request by EleosOS**
I would like to thank myself for existing and not completely giving up midway.
I would also like to note that my brain just gave up deciphering what Khaaz wrote after AxonClient.js

Needs to be checked for typos and misinformation because nobody's perfect.

### CODE REVIEW

File: src/AxonClient.js [hunk 1]
```diff
@@ -891,7 +878,7 @@ class AxonClient extends EventEmitter {
     //
 
     /**
-     * Check if the user/role/channel is guild ignored
+     * Check if the user/role/channel is ignored.
```
Comment (Khaazz) [inline src/AxonClient.js]: Specify here "ignored" means guild ignored (this guild only)

File: src/AxonClient.js [hunk 2]
```diff
@@ -905,7 +892,7 @@ class AxonClient extends EventEmitter {
     }
 
     /**
-     * Check if the module is server disabled
+     * Check if the module is disabled.
```
Comment (Khaazz) [inline src/AxonClient.js]: Specify here "disabled" means guild disabled (this guild only)

File: src/AxonClient.js [hunk 3]
```diff
@@ -917,7 +904,7 @@ class AxonClient extends EventEmitter {
     }
 
     /**
-     * Check if the command is server disabled
+     * Check if the command is disabled.
```
Comment (Khaazz) [inline src/AxonClient.js]: Specify here "disabled" means guild disabled (this guild only)

File: src/AxonClient.js [hunk 4]
```diff
@@ -1356,7 +1333,7 @@ class AxonClient extends EventEmitter {
     /**
      * Inspect method.
      * Doesn't list prefixed property and undefined property.
-     * (method took from eris)
+     * (method shamelessly copied from eris)
```
Comment (Khaazz) [inline src/AxonClient.js]: really...

File: src/Database/JsonService.js [hunk 5]
```diff
@@ -13,116 +13,107 @@ class JsonService {
...
 
     /**
-     * Update the blacklisted users
+     * Updates the blacklisted users.
      *
      * @static
-     * @param {Array<String>} blacklistedUsers - Array of blacklistedUsers
-     * @returns {Promise} AxonSchema updated
+     * @param {Array<String>} blacklistedUsers - Array of blacklisted users
+     * @returns {Promise} Updated blacklistSchema
```
Comment (Khaazz) [inline src/Database/JsonService.js]: It actually sends back the whole axonschema

File: src/Database/JsonService.js [hunk 6]
```diff
@@ -13,116 +13,107 @@ class JsonService {
...
     }
 
     /**
-     * Update the blacklisted guilds
+     * Updates the blacklisted guilds.
      *
      * @static
      * @param {Array<String>} blacklistedGuilds - Array of blacklistedUsers
-     * @returns {Promise} AxonSchema updated
+     * @returns {Promise} Updated blacklistSchema
```
Comment (Khaazz) [inline src/Database/JsonService.js]: It actually sends back the whole axonschema

File: src/Structures/Module.js [hunk 7]
```diff
@@ -15,8 +15,7 @@ import EnsureInterface from './EnsureInterface';
 
 /**
  * AxonCore - Module constructor
- * Register a Module with an Object of Commands, an Object of Events, an Object of Schemas
- * Register/unregister
+ * Register a Module with an obbject of commands, an object of events and an object of schemas.
```
Comment (Khaazz) [inline src/Structures/Module.js]: ups 2 'b's

File: src/AxonClient.js [hunk 8]
```diff
@@ -827,8 +815,7 @@ class AxonClient extends EventEmitter {
     }
 
     /**
-     * Send full help in DM
-     * Respecting permissions
+     * Send full help in DM.
```
Comment (Khaazz) [inline src/AxonClient.js]: Show here it sends the help command regarding user permissions in that guild

File: src/AxonClient.js (hunk 4 above)
Comment (EleosOS) [reply src/AxonClient.js]: It's true :man_shrugging:

**Comment by EleosOS** [comment]
Made the changes.

File: src/AxonClient.js (hunk 4 above)
Comment (Santhosh-Annamalai) [reply src/AxonClient.js]: Lul. This made my day today 😂. Wonder what would have happened if we had missed it in the PR review!

**Pull request merged**